### PR TITLE
document `sign-multisig` command in user guide

### PR DESCRIPTION
**Pull request by ratik21**

### CODE REVIEW

**Comment by amityadav0** [comment]
could you please add tests for `sign-multisig` ? i am not sure if it works, having tests before docs would be good.

**Comment by ratik21** [comment]
> could you please add tests for sign-multisig ? i am not sure if it works, having tests before docs would be good.

This is just a basic documentation task. I have tested this command and it works, but it still feels incomplete to me. As we need to "create" multisignatures as well, which we're not doing here (we're only appending). Tests are also needed. 
I added one task for test: [https://www.pivotaltracker.com/story/show/178806449](https://www.pivotaltracker.com/story/show/178806449).

**Comment by amityadav0** [comment]
okay, so it is working properly.

File: docs/guide/README.md [hunk 1]
```diff
@@ -20,6 +20,8 @@ layout: splash
 + [Test TEAL](./testing-teal.md)
 + [Best Practices](./best-practices.md)
```
Comment (robert-zaremba) [inline docs/guide/README.md:21]: Best Practices should be the last one.

**Review by robert-zaremba — APPROVED** [review]
we can create another task to create tests.

File: docs/guide/README.md (hunk 1 above)
Comment (ratik21) [reply docs/guide/README.md:21]: Alright

**Pull request merged**